### PR TITLE
Customers can receive and view messages in their inbox

### PR DESCRIPTION
**Pull request by finn-francis**

### CODE REVIEW

File: app/controllers/messages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class MessagesController < ApplicationController
+
+  def index
+    @received_messages = current_customer.received_messages
```
Comment (dmgarland) [inline app/controllers/messages_controller.rb:4]: Where is this defined? Are we missing something on the Customer model?

Comment (finn-francis) [inline app/controllers/messages_controller.rb:4]: It looks like me and Danny both needed the Customer associations so we pushed this file up in another commit

File: app/models/customer.rb [hunk 2]
```diff
@@ -10,6 +10,6 @@ class Customer < ActiveRecord::Base
   has_many :sent_messages, class_name: 'Message', foreign_key: :sender_id
 
   def messages
-    self.received_messages
+    self.messages
```
Comment (dmgarland) [inline app/models/customer.rb]: I don't get what this is supposed to do?

Comment (finn-francis) [inline app/models/customer.rb]: Removed

**Pull request merged**